controller/src/main/java/io/pravega/controller/task/Stream/StreamMetadataTasks.java

```diff
@@ -204,6 +214,12 @@ public void initializeStreamWriters(final ClientFactory clientFactory,
                                 Throwable cause = ExceptionHelpers.getRealException(e);
                                 if (cause instanceof ScaleOperationExceptions.ScalePreConditionFailureException) {
                                     response.setStatus(ScaleResponse.ScaleStreamStatus.PRECONDITION_FAILED);
+                                } else if (cause instanceof ScaleOperationExceptions.ManualScaleTimedOut) {
+                                    // Note: timedout does not mean the scale operation failed. In fact scale operation once started,
```

Comment:
@shiveshr are you going to make this changes in this PR? I'm wondering if I should review it or wait until you sort it out.